### DISCUSSION TITLE
Wie schliesst man einen HM-600/HM-800 an?

### DISCUSSION THREAD

**Discussion by dszam** [hoylabs/OpenDTU-OnBattery#285] [Q&A] [answered]
Moin,

aktuell nutze ich zwei HM-300. Einer speist direkt ein, der andere hängt an einem SmartSolar MPPT 100/20 und einer 51,2V Batterie und nutzt den Powerlimiter. Das funktioniert auch ganz wunderbar.
Ich werde demnächst meine Anlage erweitern, sodass ich den einen HM-300 durch einen HM-600 ersetzen möchte, der dann vom MPPT/Akku gespeist wird.
Ich habe noch nicht verstanden, wie ich die beiden Anschlüsse des HM-600 mit dem Akku verbinden muss. Reicht es, einen Eingang anzuschließen und den zweiten freizulassen oder werden beide Eingänge des HM parallel an den Akku angeschlossen?
Ziel ist es, dass der Powerlimiter bis zu 600W einspeist.

Vielen Dank
Viele Grüße
Dirk

**Comment by spcqike** [comment, marked as answer]
Beide parallel an den Akku. 

Entweder mit vier Kabeln oder einem entsprechend dicken und dann Y Adapter kurz vorm wechselrichter.

**Comment by E-Ulli** [comment]
Ich würde davon abraten. Habe es selber ausprobiert mit Y-Kabeln. Läuft nicht sauber, die Regelung liegt immer 30 Watt über dem maximum und beide Eingänge machen Unterschiedliche Leistung. Die Leistung an CH1 lag bei mir immer unter der von CH0. Von der Logik sollten die beide gleich sein, da kommt dann die Regelung wohl auch nicht mit klar.

**Reply by neolauren-cmd** [reply]
Und wie hast du es jetzt gemacht? Bei mir ist CH1 auch ca 30 Watt weniger als CH0und die Regelung kommt einfach damit nicht klar.

Hab 4 Kabel direkt von der 24V Batterie, aber der HM-800 regelt nur bei festeingestellten 200W richtig. Alles was darüber eingestellt ist, spinnt die Regelung. Ich dachte schon der WR ist defekt, aber anscheinend doch nicht. Kabel von drr Batterie zum WR sind Querschnitt 10qm und max 1m lang.

**Reply by schlimmchen** [reply]
Und wie sind die Verbindingen? Ich musste lernen, dass unzureichend festgeklemmte Kabelenden (Schraubverbinder, Kabelschuhe) dafür sorgen können, dass der Hoymiles komplett falsch und stark schwankend regelt. Bei mir kam der HM-1500 nicht mehr auf mehr als 1200 W und die ersten beiden Inputs waren immer deutlich weniger leistungsstark als die anderen zwei. Nachdem ich die Verdrahtung kontrolliert hatte und nur "mittelfeste" Schraubverbindungen nachgezogen habe, war das Problem verschwunden.

**Reply by E-Ulli** [reply]
Ich mache es gar nicht . Benutze nur den einen Eingang dann läufts. Das mit den Kabeln halte ich für unwahrscheinlich, ich hatte Y-Kabel verwendet. Die Akku-Spannung sielt auch eine große Rolle. Wie hoch ist die bei schlimmchen ?

**Reply by schlimmchen** [reply]
Bei uns läuft ein 48V Akku.

**Reply by neolauren-cmd** [reply]
Akkuspannung sind nur 0,1 V Unterschied. Im Moment 26,2 V und 26,1 V. Kabel sind alle fest. Plus läuft über einen Battery Protect und 1 Relais auf eine Plusschiene, wo beide Eingänge vom WR draufgehen. Der HM-800 kommt auch bei mir max auf 473 V.

**Reply by spcqike** [reply]
Das Kann schon, wie @schlimmchen geschrieben hat, mit den Verbindungen zusammen hängen. Ein kleiner übergangswiderstand von 0.1Ohm führt bei 10A (24V bei 10A sind 240W) zu 1V Spannungsabfall. Das sind immerhin 10W Verlust von zielwert. 

Ich würde auch alle verbindet und Leitungen prüfen. Die Spannungen am Eingang des WR vergleichen (unter Last). 

@schlimmchen auch wenn es nur ein workaround wäre, aber auch in so einem ungünstigen Fall wäre die vor Wochen besprochene „überregelung“ im DPL eine Lösung. (Wenn hier auch keine gute. Die Verluste müssen ja irgendwo her kommen bzw. hin gehen. Und wenn das wirklich erhöhte Widerstände an Leitungen und klemmstellen sind, sollte man die vorrangig beseitigen)

**Reply by schlimmchen** [reply]
Das sehe ich aufgrund dieser praktischen Erfahrung leider anders: Die Ausgangsleistung des Wechselrichters war, je nach eingestelltem Power Limit, deutlich höher als das Power Limit, und schwankte zudem sehr stark. Das war schon sehr merkwürdig und ich war froh, dass es nach dem Überarbeiten der Verkabelung weg war. Im Übrigen hilft es wohl leider nichts, 1800W einzustellen (oder welchen Wert auch immer), wenn der Wechselrichter nur 1500W leisten kann und dieses Limit auch schon nicht erreicht werden kann...

**Reply by E-Ulli** [reply]
Bei 26V können die 800W nicht erreicht werden da der max. Strom bei 12,5A liegt und der HM-800 dann abregelt. Maximal 325W bei 26V pro Eingang wären Theoretisch möglich. ---> spcqike die 1V differenz müsste der Hoymiles dann aber anzeigen.

**Reply by neolauren-cmd** [reply]
Und das macht er nicht. Ich hab nur 0.1 V Differenz zwischen beiden Eingängen. Daher hab ich glaub ich ein anderes Problem :-(

**Reply by spcqike** [reply]
@neolauren-cmd welche Last liegt denn in dem Fall an? 

Welche Spannungen zeigt er an, wenn du 30W Differenz hast? 

Vielleicht regelt sich der MPPT auf dem anderen Kanal einfach anders und versucht was anderes einzustellen. Aber dann hätten solche Probleme ja wohl mehrere Leute.

**Reply by neolauren-cmd** [reply]
Hab fest 220 W eingestellt. Dann fange ich mit 26,4V CH01 mit 120W  und 26,4V Ch02 mit 91W.

Sobald ich mit der Leistung höher gehe, z.B. 300W, geht CH01 auf teilweise 240W und CH02 auf 180W. Zusammen springt deann die Ausgangsleistung immer auf 110W und wieder hoch auf 430W. So geht das immer wieder im Wechsel.

Spannung ust gleichbleibend auf briden Kanälen, mit machmal 0.1V Unterschied.

**Reply by spcqike** [reply]
Aus der Entfernung schwer zu beurteilen. Dass die Leistung auf CH1 auf 240W steigt, wenn du 300W als Limit eingestellt hast, ist auch merkwürdig. normalerweise limitiert er ja jeden Eingang auf Limit/n W, also bei einem HM600/800 auf 300/2 -> 150W (ohne Berücksichtigung der Effizienz. Die tatsächliche Leistung am Eingang ist höher, aber nicht so stark)

was sagt denn deine Effizienz in den Momenten? die sollte bei >90% sein. schwankt die ebenfalls?

Ich vermute immer noch, dass es an den Verbindungen/Leitungen/Widerständen liegt. Ich meine mich zu erinnern, dass mein HM-600 keinen konstanten Strom gezogen hat, sondern leicht pulst. (hab den mal zum Testen an einem Labornetzteil betrieben, was 40V80A kann.) Ich hab den aber nur ein paar Minuten zum Testen laufen gelassen. Ich kann mir aber vorstellen, wenn der Wechselrichter das dauerhaft macht (irgendwie muss er den Strom ja begrenzen, da bietet sich ein gepulstes Signal ja an), dass dann die Spannung auch mit den pulsen einbricht, dass die Spannungsmessung im Interface aber träger ist und die kurzen Einbrüche gar nicht erfasst. Das würde man ggf. mit einem Oszi nachmessen können.

hat vielleicht jemand die Möglichkeit und Lust, das einmal nachzumessen? :)

**Reply by E-Ulli** [reply]
Hallo ich Denke das die Aussage  " normalerweise limitiert er ja jeden Eingang auf Limit/n W, also bei einem HM600/800 auf 300/2 -> 150W (ohne Berücksichtigung der Effizienz."  nicht ganz stimmt. Es wird nur die AC-Ausgangsleistung in % Limitiert. Vielleicht sollten wir mal die Wechselrichter Firmware vergleichen. Hier schonmal von meinem HM600:
Bootloader-Version	0.1.4
Firmware-Version	1.0.10
Firmware-Erstellungsdatum	2021-11-01 08:42:00
Hardware-Teilenummer	269553683
Hardware-Version	01.10

**Reply by neolauren-cmd** [reply]
Wirkungsgrad ist in dem Moment ca. 95% und geht dann wieder auf ca. 42% runter. So schwankt das die ganze Zeit. Spannung bleibt immer gleich, nur eben die Leistungssprünge.

**Reply by neolauren-cmd** [reply]
Bootloader-Version | 0.1.4
Firmware-Version | 1.0.10
Firmware-Erstellungsdatum | 2021-11-01 08:42:00

**Reply by spcqike** [reply]
@E-Ulli ich kann meinen Inverter grad icht auslesen, bin nicht vor Ort und die DTU hängt schon wieder mit fast vollem HEAP, sodass ich im LiveView nichts sehe und die API nur eingeschränkt geht 😃 

Was stimmt daran denn deiner Meinung nach nicht?

Mein HM-600 liefert bei 600W (100%) bspw. 612W AC, zieht dabei ~320W DC pro Eingang.
Wenn ich ihn auf 300W AC einstelle, zieht er ~160W DC pro Eingang. Dabei ist es egal, wieviel ein Eingang liefern könnte. (bspw. bei Verschattung eines Moduls. Das zweite Modul könnte vielleicht mehr als 160W liefern, tut es aber nicht, da jeder Eingang stumpf auf 50% seines Maximums gedrosselt wird)

Das Verhalten hatte ich in der Vergangenheit bereits dargestellt und versucht, für rein PV getriebene Systeme (ohne Akku, wie ich es bisher habe) über einen PR (https://github.com/helgeerbe/OpenDTU-OnBattery/pull/241#issuecomment-1563993866) anzupassen, sodass der Limiter das Limit höher ansetzt, wenn das gewünschte AC Limit nicht erreicht wurde, obwohl mindestens 1 Eingang auf seinem gedrosselt Maximum läuft, also mehr liefern könnte, sofern er denn dürfte.

OpenDTU-onBattery berücksichtigt aktuell nur produzierende und nicht produzierende Eingänge. Wenn bspw. an einem HM-600 nur an einem Eingang eine Batterie hängt, und man 150W AC möchte, würde openDTU-onBattery das Limit auf 300W einstellen, damit am Ende eben 150W herauskommen.

**Reply by E-Ulli** [reply]
" sodass der Limiter das Limit höher ansetzt, wenn das gewünschte AC Limit nicht erreicht wurde, obwohl mindestens 1 Eingang auf seinem gedrosselt Maximum läuft" Das erklärt die Überschreitung des Limits.  Woran erkennt die DTU ein Akkusystem und macht das nicht !?
Werden die DC ! Eingänge limitiert ? oder beide AC Ausgänge.  Sind da nicht 2 komplette HM-300 in einem HM-600 oder gibt es nur einen AC-Wandler ?

"Wenn bspw. an einem HM-600 nur an einem Eingang eine Batterie hängt, und man 150W AC möchte, würde openDTU-onBattery das Limit auf 300W einstellen, damit am Ende eben 150W herauskommen." Das hat mich am Anfang auch stark verwirrt, ist nicht so schön. Entweder Anzeige dafür anpassen oder zumindest Hinweis einblenden

**Reply by spcqike** [reply]
Vielleicht habe ich mich missverständlich ausgedrückt. Die von mir beschriebene Logik mit der Erhöhung des Limits war ein Vorschlag, der es so nicht in die Firmware geschafft hat. OpenDTU-onBattery ist wohl bisher vornehmlich für Batterie Systeme gedacht. Da ist das von mir beschriebene Szenario eher nicht möglich. (Bei reinen PV Systemen mit unterschiedlicher Ausrichtung der Module oder verschattung, wie bei mir, sehe wohl) 

Sobald man den dynamischen power limiter einschaltet, greift die darauf beschriebene Logik. Dabei prüft openDTU, ob die Eingänge eine DC Leistung liefern. Liefert ein Eingang <1 oder <2W, wird er als deaktiviert angenommen und dann wird das effektiv gesetzte Limit entsprechend erhöht, sodass die verbleibenden Eingänge die gewünschte Gesamtleistung liefern. 

Das ist meiner Meinung nach auch total sinnvoll. Stellt man sonst stur 300W ein, würde ein HM-600 ja nur 150W liefern. Obwohl ein einzelner Eingang die 300W liefern könnte, sofern das Limit entsprechend angepasst ist. 

Wie die hoymiles intern aufgebaut sind, entzieht sich meiner Kenntnis. Ob da nun mehrere HM-300 netzparallel drin sind, oder aber anderweitig mehr Leistung realisiert wird. (Vermutlich anderweitig. Es passen ja wohl keine 5x HM-300 in einen HM-1500 :D)

Soweit ich weiß setzt die HM interne Firmware aktuell kein AC Limit, sondern einfach ein gleichmäßiges Limit auf allen Eingängen. Ein AC Limit wäre zwar schöner, da verschattung etc. keine Rolle spielen würden, aber das gleichermaßen drosseln aller Eingänge ist wahrscheinlich einfacher gewesen. 

Nachtrag der Firmware:


Produktionsjahr | 2022
-- | --
Produktionswoche | 37
Modell | HM-600
Ermittelte max. Leistung | 600 W
Bootloader-Version | 0.1.4
Firmware-Version | 1.0.10
Firmware-Erstellungsdatum | 2021-11-01 08:42:00
Hardware-Teilenummer | 269553xxx
Hardware-Version | 01.10

**Comment by Manos1966** [comment]
HM-1500 inputs performance with 25V & 51V battery
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/244

Max 10A pro Eingang (9,8A eigentlich). also 10A x 51,2V = 512Watt max pro Eingang.
